content/en/storage-providers/get-started/overview/index.md

```diff
@@ -23,7 +23,7 @@ Lotus Mining for _mainnet_ has stringent minimal **[hardware requirements]({{< r
 
 The following guides are essential starting points for those willing to launch a Lotus miner:
 
-- The miner installation is covered in the [Installation guide]({{< relref "prerequisites" >}}). Once the installation is complete, the Lotus node, Lotus miner, and Lotus worker applications should all be installed.
+- The miner installation is covered in the [Installation guide]({{< relref "../../../lotus/install/prerequisites" >}}). Once the installation is complete, the Lotus node, Lotus miner, and Lotus worker applications should all be installed.
```

Comment:
I have started a audit/revision of the overview-page already where this bullet-point is removed, so maybe you could revert this change, so that branch does not need rebasing.